lib/Debugger.ProtocolHandler/Debugger.cpp

```diff
@@ -322,7 +322,7 @@ namespace JsDebug
             {
                 JsValueRef breakpoint = PropertyHelpers::GetIndexedProperty(breakpoints, index);
                 int breakpointId = PropertyHelpers::GetPropertyInt(breakpoint, PropertyHelpers::Names::BreakpointId);
-                IfJsErrorThrow(JsDiagRemoveBreakpoint(breakpointId));
+                JsDiagRemoveBreakpoint(breakpointId);
```

Comment:
I tend to prefer filtering for specific error codes rather than just ignoring all errors.